src/Symfony/Component/HttpClient/Response/AmpResponse.php

```diff
@@ -315,7 +297,7 @@ private static function followRedirects(Request $originRequest, AmpClientState $
 
             try {
                 // Discard body of redirects
-                while (null !== yield $response->getBody()->read()) {
+                while (null !== $response->getBody()->read()) {
                 }
```

Comment:
This can probably be `$response->getBody()->close()` now?

Reply:
updated thanks